plugins/steelseries/fu-steelseries-device.h

```diff
@@ -33,10 +33,8 @@ struct _FuSteelseriesDeviceClass {
 #define STEELSERIES_BUFFER_CONTROL_SIZE 64
 #define STEELSERIES_TRANSACTION_TIMEOUT 5000
 
-FuSteelseriesDeviceKind
-fu_steelseries_device_get_kind(FuSteelseriesDevice *self);
-void
-fu_steelseries_device_set_kind(FuSteelseriesDevice *self, FuSteelseriesDeviceKind kind);
+#define FU_STEELSERIES_DEVICE_FLAG_IS_DONGLE (1 << 0)
```

Comment:
I was thinking about doing the same change.

```
/**
 * FU_STEELSERIES_IS_DONGLE:
 *
 * The device is a dongle.
 *
 * Since 1.8.1
 */   
 ```
 
 and in the README.md

```
 ## Quirk Use

This plugin uses the following plugin-specific quirks:

### is dongle

The device is a dongle.

Since 1.8.1
```

I have added in the genesys plugin but ignore it if this does not make sense.

Reply:
Ya, feel free to do both, ta.